fa24/sorting-project/group4/README.md

```diff
@@ -1,4 +1,8 @@
-Current Performance:
+CSE549 Group 4
+Shreya Mahetaliya, Divya Naidu
 
-For 2^8 : Runtime cycles = 107792 cycles
+2^12: 2527585 cycles
+2^13: 5430811 cycles
+2^14: 11630768 cycles
 
+The geometric mean is approximately 11,700,000 cycles.
```

Comment:
I think the geometric mean of `2527585 5430811 11630768` should be `5,424,915`? Also please add the following contents to the README.md:

```
speedup over the current leader
which prior versions (other group's codes) you are leveraging (if any)
```